app/employeehandler.py

```diff
@@ -97,38 +98,22 @@ async def get(self, request):
 
         last_job_role = employee_info['lastRoleId']
 
-        job_role_info = employee_info.get('jobRole', "-")
-        job_role_info = [job_role_info]
+        job_role = employee_info.get('jobRole', "-")
 
-        relevant_job_role = ''
+        job_role['contractStartDate'] = format_to_uk_dates(job_role['contractStartDate'])
+        job_role['operationalEndDate'] = format_to_uk_dates(job_role['operationalEndDate'])
 
-        for job_role in job_role_info:
-            if job_role['active']:
-                relevant_job_role = job_role
-            else:
-                job_role['contractStartDate'] = format_to_uk_dates(
-                    job_role['contractStartDate'])
-                job_role['operationalEndDate'] = format_to_uk_dates(
-                    job_role['operationalEndDate'])
-
-        if relevant_job_role == '':
-            for job_role in job_role_info:
-                if job_role['uniqueRoleId'] == last_job_role:
-                    relevant_job_role = job_role
-
-        if relevant_job_role:
-            if relevant_job_role['contractStartDate']:
-                relevant_job_role['contractStartDate'] = format_to_uk_dates(
-                    relevant_job_role['contractStartDate'])
-            if relevant_job_role['contractEndDate']:
-                relevant_job_role['contractEndDate'] = format_to_uk_dates(
-                    relevant_job_role['contractEndDate'])
-            if relevant_job_role['operationalEndDate']:
-                relevant_job_role['operationalEndDate'] = format_to_uk_dates(
-                    relevant_job_role['operationalEndDate'])
+        #  if a job role is returned then format all the dates to uk
+        if job_role:
+            if job_role['contractStartDate']:
+                job_role['contractStartDate'] = format_to_uk_dates(job_role['contractStartDate'])
```

Comment:
wasn't this done a few lines above already

Reply:
This was a formatting change - instead of having each "format_to_uk_address" split between lines it was all moved to one each